doc/definition.md

```diff
@@ -490,10 +490,10 @@ This python script is saved in the file `example.py`. Then we get
 % python example.py
 Space group type: Cmce (64)
 Transformation matrix:
-    1  0  0
-    0  1  0
-    0  0  1
-Origin shift: 0.000000 0.500000 0.500000
+   1  0  0
+   0  1  0
+   0  0  1
+Origin shift: 0.000000 0.000000 0.000000
```

Comment:
@atztogo I am curious about why the origin shift has changed. Maybe related to the axis choice for orthogonal space groups based on Affine normalizers?

Reply:
I think you are correct. a(0, 0, 0) and b(1/2, 1/2, 1/2) are in the same Wyckoff set. Plus centring, we get (0, 0, 1/2).
https://www.cryst.ehu.es/cgi-bin/cryst/programs/nph-normsets?from=wycksets&gnum=64
In the current implementation of spglib, we can not distinguish them, although it would be more natural to give the origin shift (0, 0, 0).

Reply:
The following comment is unrelated to this PR. I leave it because it came to my mind.
I probably tried to find the shortest (under some condition) origin shift in this function.
https://github.com/spglib/spglib/blob/a0263dd2e22be20b39e1196a27f80c88990a09cf/src/refinement.c#L1415